packages/ember-routing/lib/location/none_location.js

```diff
@@ -23,14 +23,30 @@ export default EmberObject.extend({
   path: '',
 
   /**
-    Returns the current path.
+    Will be pre-pended to path.
+
+    @private
+    @property rootURL
+    @default '/'
+  */
+  rootURL: '/',
+
+  /**
+    Returns the current path without `rootURL`.
 
     @private
     @method getURL
     @return {String} path
   */
   getURL() {
-    return get(this, 'path');
+    let path = get(this, 'path');
+    let rootURL = get(this, 'rootURL');
+
+    // remove trailing slashes if they exists
+    rootURL = rootURL.replace(/\/$/, '');
```

Comment:
The caveat this introduces is you wouldn't be able to change the rootURL after the router setup.  Unsure if that's an actual use case or not though.  History location is also doing this as well.